recipes/invoke/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.12.2" %}
+
+package:
+  name: invoke
+  version: {{ version }}
+
+source:  
+  fn: invoke-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/invoke/invoke-{{ version }}.tar.gz
+  md5: f46192ea664325464a02f1c7f735e3cb
+
+build:
+  number: 0
+  script: python setup.py install
+  entry_points:
+    - invoke = invoke.main:program.run
+    - inv = invoke.main:program.run
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  # Python imports
+  imports:
+    - invoke
+    - invoke.parser
+    - invoke.vendor
+    - invoke.vendor.fluidity
+    - invoke.vendor.lexicon
+    - invoke.vendor.yaml2    # [py2k]
+    - invoke.vendor.yaml3    # [py3k]
+
+  commands:
+    - invoke --help
+    - inv --help
+
+about:
+  home: http://docs.pyinvoke.org
+  license: BSD
```

Comment:
His [setup.py](https://github.com/pyinvoke/invoke/blob/0f90fa5f6e35a74ff5cf364111a1663e491ba860/setup.py) only mentions `'License :: OSI Approved :: BSD License',` and `license='BSD',` 

Here's a [link](https://github.com/pyinvoke/invoke/blob/0f90fa5f6e35a74ff5cf364111a1663e491ba860/LICENSE) to his actual license

So after checking Wikipedia, I guess it's the 2-clause license version